docs/public/installation/zonemaster-engine.md

```diff
@@ -174,10 +174,10 @@ Installation will take longer time.
 * For a Perl API, see the [Zonemaster::Engine API] documentation.
 
 
-[EPEL]:                                              https://fedoraproject.org/wiki/EPEL
-[JSON-RPC API]:                                      ../using/backend/api.md
-[Mailing list zonemaster-users]:                     https://github.com/zonemaster/zonemaster/blob/master/docs/contact-and-mailing-lists.md#zonemaster-users
+[EPEL]:                                              https://docs.rockylinux.org/books/admin_guide/13-softwares/?h=epel#the-epel-repository
```

Comment:
Well, this section regards itself with Rocky Linux so I thought it would be appropriate to link to the Rocky documentation about EPEL. I just realized now that the Fedora documentation covers Rocky too, so maybe it's better to link to that after all. I'll make an update to use the Fedora link.